hydromt_wflow/workflows/soilgrids.py

```diff
@@ -165,21 +155,19 @@ def pore_size_distrution_index_layers(ds, thetas):
 
     """
 
-    da_lst = []
-    for l in range(1, 8):
-        da = xr.apply_ufunc(
-            ptf.pore_size_index_brakensiek,
-            ds["sndppt_sl" + str(l)],
-            thetas["sl" + str(l)],
-            ds["clyppt_sl" + str(l)],
-            dask="parallelized",
-            output_dtypes=[float],
-        )
-        da.name = "sl" + str(l)
-        da = da.raster.interpolate_na()
-        da_lst.append(da)
-    ds = xr.merge(da_lst)
-    return ds
+    da = xr.apply_ufunc(
```

Comment:
Use da for DataArray and ds for Dataset. To ease comprehension for the developper :)